src/main/java/cubicchunks/api/worldgen/populator/ICubicPopulator.java

```diff
@@ -72,7 +73,7 @@
      */
     @Override
     default void generate(Random random, int chunkX, int chunkZ, World world, IChunkGenerator chunkGenerator, IChunkProvider chunkProvider) {
-        for (int y = 0; y <= 16; y++) {
+        for (int y = 0; y <= Cube.SIZE; y++) {
```

Comment:
This 16 is unrelated to cube size. This is vanilla chunk height. Populator pregenerate those cubes to let default world generators do a job.

Reply:
Will be fixed asap.